artemis/src/test/resources/complete_config.toml

```diff
@@ -29,6 +29,13 @@ eth1-endpoint="http://localhost:8545"
 # output
 #Xtransaction-record-directory="/tmp/teku"
 
+# logging
+log-color-enabled=true
```

Comment:
@rolfyone Hold on your PR as toml to yaml config changes are coming, the complete_config.toml will be moved, you might be required to update these changes in complete_config.yaml.